test/gridengine/travis_script.sh

```diff
@@ -0,0 +1,51 @@
+# start master
+
+# We use a temporary directory as an export dir that will hold the shared data b
+# galaxy and gridengine:
+EXPORT=`mktemp --directory`
+chmod 777 ${EXPORT}
+docker run --hostname sgemaster --name sgemaster -d -v ${EXPORT}:/export -v $PWD/master_script.sh:/usr/local/bin/master_script.sh  manabuishii/docker-sge-master:0.1.0 /usr/local/bin/master_script.sh
+# wait to sge master
+sleep 10
+
+# start galaxy
+GALAXY_CONTAINER=quay.io/bgruening/galaxy
+GALAXY_CONTAINER_NAME=galaxytest
+GALAXY_CONTAINER_HOSTNAME=galaxytest
+
+docker run -d \
+           -e SGE_ROOT=/var/lib/gridengine \
+           --link sgemaster:sgemaster \
+           --name ${GALAXY_CONTAINER_NAME} \
+           --hostname ${GALAXY_CONTAINER_HOSTNAME} \
+           -p 20080:80  -e NONUSE="condor" \
+           -v $PWD/job_conf.xml.sge:/etc/galaxy/job_conf.xml \
+           -v ${EXPORT}:/export \
+           -v $PWD/outputhostname:/galaxy-central/tools/outputhostname \
+           -v $PWD/outputhostname.tool.xml:/galaxy-central/outputhostname.tool.xml \
+           -v $PWD/setup_tool.sh:/galaxy-central/setup_tool.sh \
+           -v $PWD/tool_conf.xml:/galaxy-central/tool_conf.xml \
+           -v $PWD/galaxy.ini:/galaxy-central/galaxy.ini \
+           -v $PWD/act_qmaster:/var/lib/gridengine/default/common/act_qmaster \
```

Comment:
Here we could include GALAXY_CONFIG_ options and maybe get rid of the modified galaxy.ini?